src/sentry/integrations/slack/integration.py

```diff
@@ -140,6 +140,21 @@ def update_organization_config(self, data: MutableMapping[str, Any]) -> None:
         self._update_and_clean_flags_in_organization_config(data=data)
         super().update_organization_config(data=data)
 
+    def get_features(self) -> dict[str, bool]:
+        """
+        Helper method to get all the features saved to this organization integration installation
+        """
+        data = self.get_config_data()
+        return data[self._FLAGS_KEY]
+
+    def has_feature(self, feature_name: str) -> bool:
+        """
+        Helper method to check if a particular feature has a flag
+        Default returns to False if the flag does not exist
+        """
+        features = self.get_features()
+        return features.get(feature_name, False)
+
```

Comment:
Should these as well as toggleableFlags be in the parent class?

Reply:
I was thinking about it, but I think I might have a better way, let me see if I can do it in this PR

Reply:
I think I'll tackle those changes in another pr to not block the GA